overview/glossary.md

```diff
@@ -53,62 +100,82 @@ A 'Line Item' is an ad server concept that contains information about how a spec
 Integrating Prebid bids with other ad sources requires a [special line item setup](/adops/adops-planning-guide.html) if you aren't using an ad server that supports Prebid natively, which most don't.
 The [Prebid Line Item Manager] can help set up GAM line items. GAM 360 has the beginnings of a [native Prebid integration](https://support.google.com/admanager/answer/12273163?hl=en).
 
-## Key Values
+### Key Values
 
 A generic way to target [line items](#line-items) in your ad manager. Prebid makes heavy use of key values to indicate the winning bidder and the winning bid price.
 
-## AMP
+## Inventory
 
-Accelerated Mobile Pages (AMP) is an alternate web platform that can speed web pages on mobile devices, but
-also limits the scripts that can be run on the page. See [Prebid's AMP page](/formats/formats.html#amp)
+A description of the context where an ad may be served.
 
-## Config ID
+### AdSlot
 
-`Prebid SDK` - when coding the mobile app for the technical implmentation of the adslot, the SDK function call
-refers to this ID that refers to an impression-level stored request on [Prebid Server]. This server-side stored
-request contains the bidders and parameters that can participate in the auction for that adslot.
+The 'hole' in the web page or mobile app where an ad may appear. This is very similar to `AdUnit`, but is
+the technical implementation defining which ad sizes and formats may appear within it.
 
-## Ad Formats
+In the "Google Publisher Tag" (GPT), the "defineSlot()" function call maps an HTML div (the technical
```

Comment:
Is AdSlot just a GPT term that we've adopted ?

Reply:
adslot is an attempt to differentiate the implementation (e.g. a div) from the metadata about ads (e.g. bidders)